dart/lib/src/browser_client.dart

```diff
@@ -17,15 +17,15 @@ import 'version.dart';
 
 SentryClient createSentryClient({
   @required String dsn,
-  Event environmentAttributes,
+  Event environment,
```

Comment:
I'm a bit lost here, why an `Event` is called `environment`?

Reply:
https://github.com/getsentry/sentry-dart/pull/108#discussion_r503902646

probably they were not exactly the same thing?

Reply:
from the doc : 
> /// [environment] contain event attributes that do not change over
  /// the course of a program's lifecycle. These attributes will be added to
  /// all events captured via this client. The following attributes often fall
  /// under this category: [Event.serverName], [Event.release], [Event.environment].

Reply:
ok so then it makes sense to keep as `environmentAttributes` till we split them up to separate fields.

Reply:
✅ reverted